build.gradle

```diff
@@ -19,6 +19,9 @@ ext {
     junitVersion = '5.5.2'
     mockitoKotlinVersion = '1.6.0'
     testContainersVersion = '1.12.0'
+    log4jVersion = '2.12.1'
```

Comment:
I am using kotlin-logging with logback

Reply:
`val logger = KotlinLogging.logger {}`

Reply:
Yeah, sorry, this is a leftover from before we had a pom file